test/UnitTest/Microsoft.AspNet.OData.Test/PublicApi/Microsoft.AspNet.OData.PublicApi.bsl

```diff
@@ -104,6 +104,7 @@ public abstract class Microsoft.AspNet.OData.EdmStructuredObject : Delta, IDynam
 	public virtual Microsoft.OData.Edm.IEdmTypeReference GetEdmType ()
 	public virtual System.Collections.Generic.IEnumerable`1[[System.String]] GetUnchangedPropertyNames ()
 	public System.Collections.Generic.Dictionary`2[[System.String],[System.Object]] TryGetDynamicProperties ()
+	public System.Collections.Generic.Dictionary`2[[System.String],[System.Object]] TryGetInstanceAnnotationDictionary ()
```

Comment:
>[System.String],[System.Object] [](start = 48, length = 31)

I thought it was IDictionary<System.String,IDictionary<System.String,System.Object>>

Reply:
`#Resolved -removed`